curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63bf47fd40599f29827f484d.md

```diff
@@ -1,15 +1,15 @@
 ---
 id: 63bf47fd40599f29827f484d
-title: Step 24
+title: Step 20
 challengeType: 0
-dashedName: step-24
+dashedName: step-20
 ---
 
 # --description--
 
-The pattern you currently have will match the exact text `hello`, which is not what you want to match. You want to look for `+`, `-`, or spaces. Replace the pattern in your `regex` variable with `\+-` to look for plus and minus characters.
+The pattern you currently have will match the exact text `"hello"`, which is not what you want to match. You want to look for `+`, `-`, or spaces. Replace the pattern in your `regex` variable with `\+-` to look for plus and minus characters.
 
-Note that you need to use the `\` to <dfn>escape</dfn> the `+`, because a `+` has a special meaning in regular expressions.
+Note that you need to use the `backslash` `\` character to <dfn>escape</dfn> the `+`, because a `+` has a special meaning in regular expressions.
```

Comment:
```suggestion
Note that you need to use the `backslash` `\` character to <dfn>escape</dfn> the `+` symbol because it has a special meaning in regular expressions.
```